do_like_javac/tools/infer.py

```diff
@@ -77,35 +77,17 @@ def get_tool_command(args, target_classpath, java_files, jaif_file="default.jaif
     if 'CLASSPATH' in os.environ:
         cp += ':' + os.environ['CLASSPATH']
 
-        # os env classpath must be added to targetclasspath for running CFI in
-        # typecheck mode
-        target_classpath += ':' + os.environ['CLASSPATH']
-        # TODO: see if this is still needed:
-        # env_classpath must also have a project's dependent jars
-        # os.environ['CLASSPATH'] = target_classpath
-
-    CFI_command += [        # '-p', # printCommands before executing
-                            '-classpath', cp,
-                            'checkers.inference.InferenceLauncher']
-
-    if not args.cfArgs == "":
-        CFI_command += [    '--cfArgs', args.cfArgs]
-
-    CFI_command += [        '--checker', args.checker,
-                            '--solver', args.solver,
-                            '--solverArgs', args.solverArgs,
-                            '--mode', args.mode,
-                            '--hacks=true',
-                            '--targetclasspath', target_classpath,
-                            '--logLevel=INFO',
-                            '--jaifFile', jaif_file]
-
-    if args.inPlace:
```

Comment:
Can you carefully go through the current logic and make sure nothing is lost?
Isn't `--inPlace` still an option?

Reply:
thanks! I think we need to undo this change.